nbs/test_apps/script_app/script.py

```diff
@@ -0,0 +1,15 @@
+# /// script
```

Comment:
I'm curious why we put these test_apps inside the notebooks folder. I think it would make them more easy to find if they're at the top level of the repo. What do you think?